pytorch_lightning/trainer/connectors/signal_connector.py

```diff
@@ -102,3 +102,12 @@ def _has_already_handler(self, signum: Signals) -> bool:
     def _register_signal(signum: Signals, handlers: HandlersCompose) -> None:
         if threading.current_thread() is threading.main_thread():
             signal.signal(signum, handlers)
+
+    @staticmethod
+    def _reset_signal(signum: Signals) -> None:
+        if isinstance(signal.getsignal(signum), HandlersCompose):
+            signal.signal(signum, signal.SIG_DFL)
+
+    def teardown(self) -> None:
+        self._reset_signal(signal.SIGTERM)
+        self._reset_signal(signal.SIGUSR1)
```

Comment:
this part is conflicting with  #10611

Reply:
is this needed for the goal of this PR ("move signal to SIGTERM")?

Reply:
No, let's use yours instead.

Reply:
I will remove this code.

Reply:
Done !